File: beginner/javascript/2311.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split("\n");

let n = parseInt(lines.shift());
        
for (let i = 0; i < n; i++) {
    let soma = 0.0;
    
    let nome = lines.shift();
    let num = parseInt(lines.shift());
    let maior = 0.0;
    let menor = 0.0;
    
    var valores = lines.shift().split(" ");
    for (var j = 0; j < 7; j++) {
        var notas = parseFloat(valores[j]);
        
        if (j == 0) {
            maior = notas;
            menor = notas;
        }
        
        if (notas > maior) {
            maior = notas;
        }
        
        if (notas < menor) {
            menor = notas;
        }
        
        soma += notas;
    }
    
    console.log(nome + " " + ((soma - (maior + menor)) * num).toFixed(2));
}
        

File: beginner/python/2311.py
n = int(input())

for i in range(n):
    soma = 0.0
    
    nome = input()
    num = float(input())
    
    maior = 0.0
    menor = 0.0
    
    notas_input = input()
    notas = [float(nota) for nota in notas_input.split()]

    for j in range(7):
        nota = notas[j]

        if j == 0:
            maior = nota
            menor = nota

        if nota > maior:
            maior = nota

        if nota < menor:
            menor = nota

        soma += nota

    resultado = ((soma - (maior + menor)) * num)

    print('%s %.2f' % (nome, resultado))
